google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploymentRunConfigurationEditorTest.java

```diff
@@ -60,6 +75,7 @@ public void testValidSelections() {
     AppEngineDeploymentConfiguration config = new AppEngineDeploymentConfiguration();
     config.setCloudProjectName("test-cloud-proj");
     config.setConfigType(ConfigType.AUTO);
+    when(cloudSdkService.hasJavaComponent()).thenReturn(true);
```

Comment:
Should we have a test that checks the validation error is shown when this is false?